File: Back/Back/Models/Local.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Back.Models
{
    public class Local
    {
        public int Id { get; set; }
        public string NombreLo { get; set; }
        public string DescripcionLo { get; set; }
        public string Categoria { get; set; }
        public string DireccionLo { get; set; }
        public string Horario { get; set; }
        public string Telefono { get; set; }
        [ForeignKey("VendedorId")]
        public Vendedor Vendedor { get; set; }

        public ICollection<Compra> Compras { get; set; }





    }
}

File: Front/src/app/interfaces/local.ts
export interface Local {
    id?:number,
    nombreLo: string;
    direccionLo: string;
    descripcionLo: string;
    vendedorid?:string,
    categoria: string;
    horario: string;
    telefono: string;
    imagenFondoUrl: string | null; // Cambiado a string | null
}
